src/components/new-request/name-check/name-check.vue

```diff
@@ -0,0 +1,809 @@
+    if (this.getIsXproMras) {
+      return 'Your name in your home jurisdiction cannot be edited'
+    }
+    return 'You can edit your name here and check it again'
+  }
+  get showDesignationSelect (): boolean {
+    return (this.$designations[this.getEntityTypeCd]?.end || false) && !this.getIsXproMras
+  }
+  get specialCharacters (): Array<string> {
+    return this.getSpecialCharacters
+  }
+  get subTitleConflict (): string {
+    if (!this.hasIssuesConflictAlert && !this.hasIssuesConflictCaution) return 'OK'
+    let message = ''
+    if (this.conflictsExactNum > 0) message += `<b>${this.conflictsExactNum}</b> exact matches`
+    else if (this.hasIssuesConflictAlert) return 'Possible issues found'
+    if (this.hasIssuesConflictCaution && message) message += `, <b>${this.conflictsSimilarNum}</b> similar names`
+    else if (this.hasIssuesConflictCaution) message += `<b>${this.conflictsSimilarNum}</b> similar names`
+    // 0 conflicts
+    return `${message} found`
+  }
+  get subTitleStructure (): string {
+    const issues = [
+      this.hasConflictsConditional,
+      this.hasConflictsRestricted,
+      this.hasDesignationsCheckUse,
+      this.hasDesignationsMismatched,
+      this.hasDesignationsMisplaced,
+      this.hasSpecialCharacters,
+      this.isMissingDescriptive,
+      this.isMissingDesignation,
+      this.isMissingDistinctive,
+      this.hasErrorsdesignations,
+      this.hasErrorsConflictsRestricted,
+      this.hasErrorsStructure
+    ].filter(function (issue) { return issue })
+    if (issues.length > 0) {
+      return `<b>${issues.length}</b> possible issues found`
+    }
+    return 'OK'
+  }
+  get tabIconConflict (): { color: string, icon: string } {
+    if (this.hasIssuesConflictAlert) return { color: null, icon: 'mdi-alert-octagon' }
+    else if (this.hasIssuesConflictCaution) return { color: null, icon: 'mdi-alert' }
+    return { color: null, icon: 'mdi-check' }
+  }
+  get tabIconStructure (): { color: string, icon: string } {
+    if (this.hasIssuesStructureAlert) {
+      return { color: null, icon: 'mdi-alert-octagon' }
+    }
+    if (this.hasIssuesStructureCaution) {
+      return { color: null, icon: 'mdi-alert' }
+    }
+    return { color: null, icon: 'mdi-check' }
+  }
+  get tabIconVerdict (): { color: string, icon: string } {
+    if (this.hasIssuesStructureAlert || this.hasIssuesConflictAlert) {
+      return { color: 'red darken-2', icon: 'mdi-alert-octagon' }
+    }
+    if (this.hasIssuesConflictCaution || this.hasIssuesStructureCaution) {
+      return { color: 'orange darken-2', icon: 'mdi-alert' }
+    }
+    return { color: 'green darken-2', icon: 'mdi-check-circle' }
+  }
+  get tradeMarkDBLink (): string {
+    // TODO: make it a config var?
+    return 'http://www.strategis.ic.gc.ca'
+  }
+  back () {
+    this.setActiveComponent('Tabs')
+  }
+  checkAgain () {
+    this.startAnalyzeName(null)
+  }
+  clearError (errorType: NameCheckErrorType) {
+    console.log(errorType)
```

Comment:
Same as above. Definitely has value for development but i'm not sure if we would want to commit this.

Reply:
sorry will take these out

Reply:
I confirmed this is removed, so resolving this conversation.